src/utils/error-messages.ts

```diff
@@ -0,0 +1,34 @@
+enum Errors {
+  NoToken = 'No token, authorization denied',
+  InvalidToken = 'Token is not valid',
+
+  InvalidValue = 'err_invalid_value_for_',
+  MiddlewareException = 'Exception caught in error middleware - ',
```

Comment:
This `enum` should contains only "bad request errors" that will return to client
logs messages should be in their place (where you log the message)